src/controllers/uploader.ts

```diff
@@ -0,0 +1,45 @@
+import { Request, Response, NextFunction } from 'express'
+import aws from 'aws-sdk'
+
+import {
+  NotFoundError,
+  UnauthorizedError,
+  InternalServerError,
+  BadRequestError,
+} from '../helpers/apiError'
+aws.config.update({
+  region: 'us-east-1',
+  accessKeyId: process.env.AWSAccessKeyId,
+  secretAccessKey: process.env.AWSSecretKey,
```

Comment:
Can you rename the env variables also:
`AWS_ACCESS_KEY`
`AWS_SECRET_KEY`

Reply:
Done